src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -176,7 +189,9 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
   }
 
   private def persistStatus(calls: Traversable[Call], callStatus: CallStatus): Future[Unit] = {
-    executionStore ++= calls.map { _ -> callStatus}.toMap
+    executionStore ++= calls map {
+      CallKey(_, None, None) -> callStatus
+    }
```

Comment:
Yep. A lot of `WorkflowActor` needs new plumbing to stop using just a `Call`. For example `persistStatus` should be using `Traversable[CallKey]` that also contains the index.